hopsworks-common/src/main/java/io/hops/hopsworks/common/util/Settings.java

```diff
@@ -3079,4 +3081,8 @@ public synchronized String getJWTSigningKeyName() {
     checkCache();
     return JWT_SIGNING_KEY_NAME;
   }
+
+  public String getHiveSiteSparkHdfsPath() {
+    return "hdfs:///user/spark/hive-site.xml";
```

Comment:
@Limmen remove the hardcoded `spark` user in the path and replace it with `getSparkUser()`

Reply:
Will do, wasn't aware of `getSparkUser()`

Reply:
Done